### ISSUE TITLE
[Request] Custom scenes and playlists

### ISSUE THREAD

**Issue by fumei-dev** [stashapp/stash#226]
Markers are nice and all, but it would be really nice to be able to create custom scenes. As for the name of these custom scenes, I think "moments" or "clips" would be pretty appropriate for this. How I imagine that this would work would be that you go to the Moments section and either play a moment or press "Create Moment". When creating a moment, you select a file and set the start & stop time and have the ability to add more sections from the same or different scenes, as well as manage the order that they will play in. 

The playlist idea is pretty self explanitory, but it would be cool to be able to add videos as well as "moments" and to have a shuffle option when veiwing them.

**Comment by echo6ix** [comment]
Cool idea.

Not trying to hijack the thread, but this could fit into the channels paradigm suggested in #115 right [here](https://github.com/stashapp/stash/issues/115#issuecomment-559706125).

TLDR The _channels_ concept could act as a user-defined recursive object containing Stash items (studios, performers, tags, markers, scenes) and playlists. Rather than bogging Stash down with a lot of rigid sections like studios, dvds, custom scenes, etc, it could potentially all be done in something like _channels_.

**Comment by Leopere** [comment]
I don't know if we're planning on making a video editor for StashApp however playlists are something that's been alluded to for a while so I imagine that portion of this issue can likely be slotted for the dev cycle.  We already have a version where you can search for tags and random play from that search function etc.

**Comment by neru2132** [comment]
I think #5311 makes the proposed clips feature almost entirely redundant.

While #309 is closed, there is currently no way of having persistent custom playlists where the order can be freely defined and scenes can be mixed with markers. This could be its own feature request, however, so this might be a candidate for a closeable issue.
